tests/test_core.py

```diff
@@ -5202,6 +5202,10 @@ def test_fs_nodefs_home(self):
     self.emcc_args += ['-lnodefs.js']
     self.do_runf(test_file('fs/test_nodefs_home.c'), 'success', js_engines=[config.NODE_JS])
 
+  def test_fs_nodefs_errno_closure(self):
+    self.emcc_args += ['-lnodefs.js', '--closure=1']
+    self.do_runf(test_file('fs/test_nodefs_errno_closure.c'), 'success', js_engines=[config.NODE_JS])
```

Comment:
I wonder if we can get test coverage for this simply by adding `self.maybe_closure()` to some/all of the exists nodefs tests?  (i.e. avoid adding a new test).

Reply:
BTW, even we do add this new test that way to get closure coverage in the core tests in this file is to add `self.maybe_closure()` rather than ` '--closure=1'` directly.

Reply:
Yes it looks like for example you can add `self.maybe_closure()` to the existing `test_fs_nodefs_home` test

Reply:
I'm unsure if `test_fs_nodefs_home` or any existing test would catch this specific case, as this issue happens only when an error is thrown by Node and it seems all these tests expect no error to happen (and therefore we never need to do that problematic dynamic lookup).

Reply:
Hmm.. if none of the existing tests are testing error conditions that sounds pretty bad.. maybe find the existing test that uses exercises `stat` and add some error condition testing to it?

Reply:
Those existing tests do look quite spotty!

How about you just add a new assert to `./tests/fs/test_nodefs_rw.c` ?  Just try to open a non-existent file and check the error code?

Reply:
I've replaced my test case by 2 additional asserts in 99b9306c9ab2d06388ec28ec0e6019e595859ed9, seems to cover the case nicely